cms/src/components/groups/GroupEditor.vue

```diff
@@ -389,7 +402,13 @@ const saveChanges = async () => {
                     <LBadge v-if="isLocalChange(group._id) && !isConnected" variant="warning">
                         Offline changes
                     </LBadge>
-
+                    <LButton
+                        v-if="groups && groups.length > 0"
+                        variant="secondary"
+                        @click="duplicateGroup"
+                    >
+                        Duplicate
+                    </LButton>
```

Comment:
This button feels a bit prominent right now, whereas I think it will not be the most important action on the page. It also clashes with the "save changes" button if there are changes:

<img width="1355" alt="image" src="https://github.com/bccsa/luminary/assets/39056299/190b3f0a-c2cc-4dd1-9bd5-981dabcee421">


I propose two changes:
- Only show the button when the group accordion is opened
- Instead of a text button, make it just an icon button, and make it tertiary, so it doesn't stand out as much.